src/main/kotlin/org/rust/ide/annotator/RsSyntaxErrorsAnnotator.kt

```diff
@@ -168,6 +169,17 @@ private fun checkValueParameter(holder: AnnotationHolder, param: RsValueParamete
     }
 }
 
+private fun checkTypeParameterList(holder: AnnotationHolder, element: RsTypeParameterList) {
+    val lifetimeParams = element.lifetimeParameterList
+    if (lifetimeParams.isEmpty()) return
+    val startOfTypeParams = element.typeParameterList.minBy { it.textOffset }?.textOffset ?: return
```

Comment:
Can we be sure that list of children in psi element has same order that in code?
If yes then `.firstOrNull()?.textOffset` is the best variant.